Services/CustomersService.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace PSP.Services
 {
-    public class CustomersService : CrudEntityService<Customer, CustomerLogin>, ICustomersService
+    public class CustomersService : CrudEntityService<Customer, CustomerCreate>, ICustomersService
```

Comment:
https://github.com/AusteZ/PSP/pull/14/files#r1450131123

Reply:
can you elaborate? :D